packages/vite/src/client/client.ts

```diff
@@ -432,14 +432,7 @@ export function createHotContext(ownerPath: string): ViteHotContext {
       }
     },
 
-    acceptDeps() {
-      throw new Error(
-        `hot.acceptDeps() is deprecated. ` +
-          `Use hot.accept() with the same signature instead.`
-      )
-    },
-
-    dispose(cb) {
+    dispose(cb: (data: any) => void) {
```

Comment:
I think we should add types in a separate PR, so that the history is more clean

Reply:
I think that comes from rebase. already removed.